server/website/website/settings/constants.py

```diff
@@ -4,10 +4,22 @@
 # Copyright (c) 2017-18, Carnegie Mellon University Database Group
```

Comment:
All settings that you moved to the Session model can now be removed from this file.